src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String TYPE_SUPPLY = "supply";
+    private static final String TYPE_BUY = "buy";
+    private static final String TYPE_RESULT = "result";
+    private static final String SPECIFIC_CHARACTER = ",";
+    private static final int WORD = 0;
```

Comment:
Нуль - це не якесь слово, а індекс, під яким знаходиться Тип операції. Змінним завжди треба давати осмислені імена
```suggestion
    private static final int OPERATION_TYPE_INDEX = 0;
```